dayrepo/snippets/models.py

```diff
@@ -55,21 +77,194 @@ class Meta(object):
     # 項目作成
 
     id = models.AutoField(
-        verbose_name="車両ID", primary_key=True, editable=True, blank=False, null=False
+        verbose_name="車両id", 
+        primary_key=True, 
+        editable=True, 
+        blank=False, 
+        null=False
     )
     # 車両番号はそのまま保存すると「名古屋　１２３　た　４５６７」のようになってしまうため、空白をハイフン埋めする方針でいく
     # TODO: バリデーションの実装 issue -> https://github.com/bob-g12/dayrepo-for-transportation/issues/15
 
     vehicle_number = models.CharField(
-        verbose_name="車両番号", max_length=15, blank=False, null=False
+        verbose_name="車両番号", 
+        max_length=15, 
+        blank=False, 
+        null=False
+    )
+    now_mileage = models.IntegerField(
+        verbose_name="走行距離", 
+        blank=False, 
+        null=False
+    )
+
+    def __str__(self):
+
+        return str(self.id)
+
+
+class Snippet(models.Model):
+
+    class Meta(object):
+
+        # 作成されるテーブル名を指定
+        db_table = "snippets"
+        # 管理画面でのモデルの名称
+        verbose_name = "日報"
+
+    # 項目作成
+
+    id = models.AutoField(
+        verbose_name="日報id",
+        primary_key=True,
+        editable=True,
+        blank=False,
+        null=False,
+    )
+    duties_id = models.ForeignKey(
+        'Dutiestrouble', 
```

Comment:
くっそ細かいけど。migrateもワンチャンうまくできてない可能性があるから、コマンドの打ち直しもよろしく
```suggestion
        'DutiesTrouble', 
```